src/widgets/log_window/__init__.py

```
@@ -0,0 +1,22 @@
+from PyQt5.QtWidgets import QDialog, QVBoxLayout, QPlainTextEdit
+from PyQt5.QtCore import Qt
```

Ruff (0.12.2)

`PyQt5.QtCore.Qt` imported but unused

(F401)